src/Serilog.Sinks.Email/Sinks/Email/EmailSink.cs

```diff
@@ -104,46 +69,26 @@ public async Task EmitBatchAsync(IEnumerable<LogEvent> events)
             var subject = new StringWriter();
             _subjectLineFormatter.Format(events.OrderByDescending(e => e.Level).First(), subject);
 
-            var mailMessage = new MailMessage
+            var email = new Sinks.Email.Email
             {
-                From = new MailAddress(_connectionInfo.FromEmail),
+                From = _connectionInfo.FromEmail,
                 Subject = subject.ToString(),
                 Body = payload.ToString(),
-                BodyEncoding = Encoding.UTF8,
-                SubjectEncoding = Encoding.UTF8,
-                IsBodyHtml = _connectionInfo.IsBodyHtml
+                IsBodyHtml = _connectionInfo.IsBodyHtml,
+                Tos = _connectionInfo.ToEmail.Split(";".ToCharArray(), StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Should split by both `;` and `,`

https://github.com/serilog/serilog-sinks-email/blob/dev/src/Serilog.Sinks.Email/Sinks/Email/EmailConnectionInfo.cs#L66